README.md

```diff
@@ -44,54 +44,54 @@ You are in charge of your data and customizations.
+If your Pull Request is accepted as it fixes a bug, adds functionality, or makes AppFlowy's codebase significantly easier to use or understand, **Congratulations!** If your administrative and managerial work behind the scenes sustains the community, **Congratulations!** You are now an official contributor to AppFlowy. Get in touch with us ([link](https://tally.so/r/mKP5z3)) to receive the very special Contributor T-shirt!
 Proudly wear your T-shirt and show it to us by tagging [@appflowy](https://twitter.com/appflowy) on Twitter.
 
+## Join the community to build AppFlowy together
 
-## Join the community to build AppFlowy together!
 <a href="https://github.com/AppFlowy-IO/AppFlowy/graphs/contributors">
   <img src="https://contrib.rocks/image?repo=AppFlowy-IO/AppFlowy" />
 </a>
 
 ## Why Are We Building This?
 
-Notion has been our favorite project and knowledge management tool in recent years because of its aesthetic appeal and functionality. Our team uses it daily, and we are on its paid plan. However, as we all know Notion has its limitations. These include weak data security and poor compatibility with mobile devices. Likewise, alternative collaborative workplace management tools also have their constraints.
+Notion has been our favourite project and knowledge management tool in recent years because of its aesthetic appeal and functionality. Our team uses it daily, and we are on its paid plan. However, as we all know, Notion has its limitations. These include weak data security and poor compatibility with mobile devices. Likewise, alternative collaborative workplace management tools also have their constraints.
 
-The limitations we encountered using these tools rooted in our past work experience with collaborative productivity tools lead to our firm belief that there is, and will be a glass ceiling on what's possible in the future for tools like Notion. This emanates from these tools probable struggles to scale horizontally at some point. It implies that they will likely be forced to prioritize for a proportion of customers whose needs can be quite different from the rest. While decision-makers want a workplace OS, the truth is that it is not very possible to come up with a one-size fits all solution in such a fragmented market.
+The limitations we encountered using these tools rooted in our past work experience with collaborative productivity tools led to our firm belief that there is, and will be, a glass ceiling on what's possible in the future for tools like Notion. This emanates from these tools probable struggles to scale horizontally at some point. They will likely be forced to prioritize a proportion of customers whose needs differ from the rest. While decision-makers want a workplace OS, the truth is that it is impossible to come up with a one-size fits all solution in such a fragmented market.
```

Comment:
```suggestion
The limitations we encountered using these tools and our past work experience with collaborative productivity tools have led to our firm belief that there is a glass ceiling on what's possible for these tools in the future. This emanates from the fact that these tools will probably struggle to scale horizontally at some point and be forced to prioritize a proportion of customers whose needs differ from the rest. While decision-makers want a workplace OS, it is impossible to come up with a one-size fits all solution in such a fragmented market.
```

This paragraph sounds off and there are some parts I don't completely understand. It's just a suggestion, so do with it what you will.

Reply:
I love ❤️ this suggestion, thanks.